lms/templates/problem.html

```diff
@@ -5,13 +5,13 @@
 %>
 
 <%namespace name='static' file='static_content.html'/>
-<h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
+<h3 class="hd hd-2 problem-header" id="${ short_id }-problem-title" aria-describedby="${ id }-problem-progress" tabindex="-1">
   ${ problem['name'] }
 </h3>
 
 <div class="problem-progress" id="${ id }-problem-progress"></div>
 
-<div class="problem" aria-labelledby="${ id }-problem-title">
+<div class="problem" aria-labelledby="${ short_id }-problem-title">
```

Comment:
@muzaffaryousaf while you are making this change, can you also handle https://openedx.atlassian.net/browse/TNL-5671? I started to create a branch for it myself, but realized I needed to change id to short_id, and then recalled you had this conversion in progress in this PR.

As TNL-5671 states, the aria-labelledby needs to move from this div to the div in problem_ajax.html (and role="group" added).